docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -90,7 +90,7 @@ A collection skeleton is a directory that looks like a collection directory but
 
 An example ``galaxy.yml.j2`` file that accepts an optional dictionary variable ``dependencies`` could look like this:
 
-.. code-block:: yaml
+.. code-block:: ini
```

Comment:
I think this should be `jinja`, since it is a Jinja template (that will end up as YAML, but right now is not YAML). INI is definitely wrong here.